src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -254,6 +285,31 @@ The time ends when:
 * All response data has been sent.
 * The context data structures for the request are being disposed.
 
+## Experimental support for gRPC requests
+
+gRPC instrumentation can be enabled by setting
+`OTEL_DOTNET_EXPERIMENTAL_ASPNETCORE_ENABLE_GRPC_INSTRUMENTATION` flag to
+`True`. The flag can be set as an environment variable or via IConfiguration as
```

Comment:
nit: Use the same casing throughout the doc. The code snippet below uses `true`.